tests/test_readme.py

```diff
@@ -0,0 +1,49 @@
+import association_measures.frequencies as fq
+import association_measures.measures as am
+from pandas import read_csv
+
+
+def test_input():
```

Comment:
These tests don't assert anything?

Reply:
this is my way of creating the output for the README. it doesn't test anything in addition to all the other tests. you can change it if you like!